node-runner-cli/api/CoreApiHelper.py

```diff
@@ -60,14 +63,38 @@ def network_status(self, print_response=False):
             except ApiException as e:
                 Helpers.handleApiException(e)
     
-    def engine_configuration(self):
+    def engine_configuration(self, print_response=False):
         with core_api.ApiClient(self.system_config) as api_client:
             api_client = self.set_basic_auth(api_client, "admin", "admin")
             try:
                 api = engine_api.EngineApi(api_client)
                 response: EngineConfigurationResponse = api.engine_configuration_post(
                     EngineConfigurationRequest(self.network_configuration().network_identifier))
-                return response
+                return self.handle_response(response, print_response)
+            except ApiException as e:
+                Helpers.handleApiException(e)
+    
+    def vote(self, print_response=False):
+        with core_api.ApiClient(self.system_config) as api_client:
+            api_client = self.set_basic_auth(api_client, "superadmin", "superadmin")
+            try:
+                api = key_api.KeyApi(api_client)
+                request = UpdateVoteRequest(network_identifier=self.network_configuration().network_identifier)
+                Helpers.print_request_body(request, "/key/vote")
+                response: UpdateVoteResponse = api.key_vote_post(request)
+                return self.handle_response(response, print_response)
```

Comment:
What do you think about telling the user the vote for fork X was submitted successfully, and explain when they may need to manually revert? IE roughly what we had in the NOTICE on the last commit?

Reply:
I added the previous NOTICE with some minor changes
![image](https://user-images.githubusercontent.com/3509654/163783623-8cdd90d4-e9bf-4a98-8dd2-7a738c53f727.png)

Reply:
Nice :)